apps/web/src/components/Feed/Socialize/CommentBox/CommentBoxIcon.tsx

```diff
@@ -110,29 +109,27 @@ export function CommentBoxIcon({ queryRef, eventRef }: Props) {
 
       <AnimatePresence>
         {showCommentBox && (
-          <FloatingOverlay style={{ zIndex: 10 }}>
```

Comment:
Why did we get rid of the Floating overlay?

Reply:
It makes the screen flickering

https://user-images.githubusercontent.com/4480258/226788490-f225141d-dc97-43db-bb39-1009edf8dc8b.mp4